source/client/gui/screens/StartMenuScreen.cpp

```diff
@@ -13,7 +13,7 @@
 #include "SelectWorldScreen.hpp"
 #include "JoinGameScreen.hpp"
 
-#if defined(_WIN32) || (defined(TARGET_OS_MAC) && TARGET_OS_IPHONE == 0)
+#if defined(USE_SDL) || defined(_WIN32) || (defined(TARGET_OS_MAC) && TARGET_OS_IPHONE == 0)
```

Comment:
`USE_SDL` might be defined on Android.  Now, I don't know why Android games don't give you a quit option, but it's probably best to imitate them.

Reply:
Fixed.